.travis.yml

```diff
@@ -19,24 +19,24 @@ matrix:
   include:
     - os: linux
       compiler: gcc
-      env: REQUIRED_ONLY=--required-only BUILD_TYPE=Debug CODECOV=ON COMPILER=GCC CATKIN_CONFIG_OPTIONS="-DCMAKE_BUILD_TYPE=Debug -DTREAT_WARNINGS_AS_ERRORS=ON" DOWNLOAD_TAGFILES=OFF
+      env: BUILD_NAME=TRUSTY_DEBUG REQUIRED_ONLY=--required-only BUILD_TYPE=Debug COMPILER=GCC CATKIN_CONFIG_OPTIONS="-DCMAKE_BUILD_TYPE=Debug -DTREAT_WARNINGS_AS_ERRORS=ON" DOWNLOAD_TAGFILES=OFF
     - os: linux
       compiler: gcc
-      env: REQUIRED_ONLY=--required-only BUILD_TYPE=Release CODECOV=OFF COMPILER=GCC CATKIN_CONFIG_OPTIONS="-DCMAKE_BUILD_TYPE=Release -DTREAT_WARNINGS_AS_ERRORS=ON" DOWNLOAD_TAGFILES=ON
+      env: BUILD_NAME=TRUSTY_RELEASE REQUIRED_ONLY=--required-only BUILD_TYPE=Release COMPILER=GCC CATKIN_CONFIG_OPTIONS="-DCMAKE_BUILD_TYPE=Release -DTREAT_WARNINGS_AS_ERRORS=ON" DOWNLOAD_TAGFILES=ON
     - os: linux
       compiler: gcc
-      env: BUILD_TYPE=Debug CODECOV=ON COMPILER=GCC CATKIN_CONFIG_OPTIONS="-DCMAKE_BUILD_TYPE=Debug -DTREAT_WARNINGS_AS_ERRORS=ON" DOWNLOAD_TAGFILES=OFF
+      env: BUILD_NAME=TRUSTY_FULL_DEBUG BUILD_TYPE=Debug COMPILER=GCC CATKIN_CONFIG_OPTIONS="-DCMAKE_BUILD_TYPE=Debug -DTREAT_WARNINGS_AS_ERRORS=ON" DOWNLOAD_TAGFILES=OFF
     - os: linux
       compiler: gcc
-      env: BUILD_TYPE=Release CODECOV=OFF COMPILER=GCC CATKIN_CONFIG_OPTIONS="-DCMAKE_BUILD_TYPE=Release -DTREAT_WARNINGS_AS_ERRORS=ON" DOWNLOAD_TAGFILES=ON
+      env: BUILD_NAME=TRUSTY_FULL_RELEASE BUILD_TYPE=Release COMPILER=GCC CATKIN_CONFIG_OPTIONS="-DCMAKE_BUILD_TYPE=Release -DTREAT_WARNINGS_AS_ERRORS=ON" DOWNLOAD_TAGFILES=ON
     - os: osx
       osx_image: xcode9.3
       compiler: clang
-      env: BUILD_TYPE=Debug CODECOV=OFF COMPILER=CLANG CATKIN_CONFIG_OPTIONS="-DCMAKE_BUILD_TYPE=Debug -DTREAT_WARNINGS_AS_ERRORS=ON"
+      env: BUILD_NAME=XCODE93_DEBUG BUILD_TYPE=Debug COMPILER=CLANG -DTREAT_WARNINGS_AS_ERRORS=ON"
```

Comment:
There's a trailing `"` here from the definition of `CATKIN_CONFIG_OPTIONS` that used to be here. How should we set `TREAT_WARNINGS_AS_ERRORS` correctly?

Reply:
This is just an oversight. `TREAT_WARNINGS_AS_ERRORS` should be set in `script_macos.sh`.